content/en/developers/metrics/agent_metrics_submission.md

```diff
@@ -23,7 +23,7 @@ For example, submitting samples 2, 3, 6, 7 sends a value of 5 (7-2) during the f
 
 Function template:
 ```python
-self.monotonic_count(name, value, tags=None, hostname=None, device_name=None)
+self.monotonic_count(name="<METRIC_NAME>",value=<METRIC_VALUE>, tags=None, hostname=None, device_name=None)
```

Comment:
let's not make this change, as in python this also means that `name` and `value` are optional, and that their default values are literally `"<METRIC_NAME>"` and `"<METRIC_VALUE>"` respectively. Let's keep it to:

`self.monotonic_count(name, value, tags=None, hostname=None, device_name=None)`